lib/tmuxinator/config.rb

```diff
@@ -62,7 +62,7 @@ def configs
         end
       end
 
-      def validate(name, custom_name = nil)
+      def validate(name, options={})
```

Comment:
Assignment Branch Condition size for validate is too high. [18.68/15]<br>Surrounding space missing in default value assignment.